OpenRA.Mods.Common/Traits/Render/RenderRangeCircle.cs

```diff
@@ -27,6 +29,21 @@ class RenderRangeCircleInfo : ITraitInfo, IPlaceBuildingDecorationInfo, IRuleset
 		[Desc("Range to draw if no armaments are available")]
 		public readonly WDist FallbackRange = WDist.Zero;
 
+		[Desc("Which circle to show. Valid values are `Maximum`, and `Minimum`")]
+		public readonly RangeCircleMode RangeCircleMode = RangeCircleMode.Maximum;
+
+		[Desc("Color of the circle")]
+		public readonly Color Color = Color.Yellow;
+
+		[Desc("Alpha value of the color of the circle")]
```

Comment:
You shouldn't need this, Color has the alpha values definable on it's own, so you should directly take it from there and edit the default values to four-byte-hex format instead of the predefined colors.